rosdep/python.yaml

```diff
@@ -6496,6 +6503,19 @@ python3-pyqtgraph:
   fedora: [python3-pyqtgraph]
   gentoo: [dev-python/pyqtgraph]
   ubuntu: [python3-pyqtgraph]
+python3-pystemd:
```

Comment:
I am actually happy to leave all the pip stuff out of this target as we won't be using it. 
Not sure how to declare this target then.

Reply:
So after my comment above about the naming, The naming is precisely designed to prevent us from overlooking a conflicting rule like this one. 

In thinking about it I believe that this key as written is actually what you want to use. The pip rules will install to your default python version for the rosdistro. Which for the older platforms Melodic and older is Python 2. If you're using Python 2 with Noetic or newer the default rosdep resolutions will not work.

Reply:
I get the whole conflicting part with future updates now I guess.

Rewrote the PR to comply!